yamllint/config.py

```diff
@@ -96,6 +102,15 @@ def parse(self, raw_content):
             self.ignore = pathspec.PathSpec.from_lines(
                 'gitwildmatch', conf['ignore'].splitlines())
 
+        if 'yaml-files' in conf:
+            if not (isinstance(conf['yaml-files'], list)
+                    and all(isinstance(i, str) for i in conf['yaml-files'])):
+                raise YamlLintConfigError(
+                                        'invalid config: yaml-files '
+                                        'should be a list of file patterns')
```

Comment:
This is not very Pythonic, I'm surprised the linter doesn't complain about that.
Can you please use either:
```python
                raise YamlLintConfigError('invalid config: yaml-files should '
                                          'be a list of file patterns')
```
or:
```python
                raise YamlLintConfigError(
                    'invalid config: yaml-files should be a list of file '
                    'patterns')
```

Reply:
done